package.json

```diff
@@ -114,20 +116,26 @@
     "webpack-merge": "5.8.0"
   },
   "scripts": {
-    "build": "node ./node_modules/webpack/bin/webpack.js --mode production",
+    "build": "yarn build:preglean && yarn build:gleanfiles && node ./node_modules/webpack/bin/webpack.js --mode production",
     "build:dev": "node ./node_modules/webpack/bin/webpack.js --mode development",
+    "build:preglean": "apt-get update && apt-get install python3-venv -y",
+    "build:pregleansudo": "sudo apt-get update && sudo apt-get install python3-venv",
```

Comment:
I tried adding python to the docker file originally and it appears we don't use a single docker file for all the different steps.  I don't really understand the whole setup.  It got funky because one of the tests ran on a system where I couldn't install without sudo and another I didn't have access to run sudo.